### DISCUSSION TITLE
Preserve EXIF data in TIFF image

### DISCUSSION THREAD

**Discussion by konradmai** [ImageMagick/ImageMagick#7547] [Help]
### ImageMagick version

7.1.1 Q16-HDRI

### Operating system, version and so on

Windows 10

### Description

We would like to compress TIFF images with LZW compression and preserve the EXIF Data. Is this possible? How can I achieve this?
If I compress with the "magick" tool, then the EXIF data will be removed.

Thanks and best regards,
Konrad

**Comment by fmw42** [comment]
Currently you will have to use Exiftool to copy the EXIF data.
